docs/3.0/deploy/infrastructure-concepts/workers.mdx

```diff
@@ -102,6 +102,41 @@ Workers have two statuses: `ONLINE` and `OFFLINE`. A worker is online if it send
 If a worker misses three heartbeats, it is considered offline. By default, a worker is considered offline a maximum of 90 seconds 
 after it stopped sending heartbeats, but you can configure the threshold with the `PREFECT_WORKER_HEARTBEAT_SECONDS` setting.
 
+### Worker logs
+<Warning>This feature is experimental as of Prefect version 3.X.X</Warning>
+
+<span class="badge cloud"></span>Workers can now send logs directly to Prefect Cloud. To enable this feature:
+
+1. Set the environment variable:
+
+```bash
+PREFECT_EXPERIMENTS_WORKER_LOGGING_TO_API_ENABLED=True
+```
+
+Once enabled:
+- All worker logs are automatically sent to the Prefect Cloud API
+- Logs are accessible through both the Prefect Cloud UI and API
+- Each flow run log includes a direct link to its associated worker's logs
+
+### Worker details
+<Warning>This feature is experimental as of Prefect version 3.X.X</Warning>
+
+<span class="badge cloud"></span> The Worker Details page shows you three key areas of information:
+
+- Runtime Information
+  - Current Prefect version
+  - Active worker configuration
+
+- Integrations
+  - Installed packages (e.g., `prefect-aws`, `prefect-gcp`)
+  - Integration-specific configurations and status if available
```

Comment:
I'd leave this out until workers actually show more than the integration names and versions.

Reply:
The whole thing or just the second bullet?

Reply:
After thinking about it, I'd collapse them -- this is what I mean: https://github.com/PrefectHQ/prefect/pull/15920#discussion_r1828443266